### ISSUE TITLE
KeyError: 'MemUsed'

### ISSUE THREAD

**Issue by pawamoy** [qtile/qtile#1435]
```
Traceback (most recent call last):
  File "/media/data/dev/forks/qtile/scripts/../bin/qtile", line 33, in <module>
    qtile.main()
  File "/media/data/dev/forks/qtile/libqtile/scripts/qtile.py", line 130, in main
    q = make_qtile()
  File "/media/data/dev/forks/qtile/libqtile/scripts/qtile.py", line 124, in make_qtile
    state=options.state,
  File "/media/data/dev/forks/qtile/libqtile/core/manager.py", line 193, in __init__
    self._process_screens()
  File "/media/data/dev/forks/qtile/libqtile/core/manager.py", line 404, in _process_screens
    self.groups[i],
  File "/media/data/dev/forks/qtile/libqtile/config.py", line 255, in _configure
    i._configure(qtile, self)
  File "/media/data/dev/forks/qtile/libqtile/bar.py", line 204, in _configure
    i._configure(qtile, self)
  File "/media/data/dev/forks/qtile/libqtile/widget/base.py", line 440, in _configure
    _TextBox._configure(self, qtile, bar)
  File "/media/data/dev/forks/qtile/libqtile/widget/base.py", line 361, in _configure
    self.formatted_text,
  File "/media/data/dev/forks/qtile/libqtile/widget/base.py", line 317, in formatted_text
    return self.fmt.format(self._text)
KeyError: 'MemUsed'
```

I believe the `Memory` widget is responsible.

**Comment by pawamoy** [comment]
Hmm in fact it was because the xephyr script tries to load my own config file. Trying with the default config this traceback disappear.

However I'm not sure why the exception happens in xephyr and not in my main X display.

**Comment by pawamoy** [comment]
Should have been fixed by #1481. If not, still closing since it was a very specific scenario and had no impact at all. Reopen if needed.
